content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: One Year of Excalidraw
+date: 2020-12-25
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Some words from the big boss [@vjeux](https://twitter.com/vjeux)
+
+> Bla bla bla
+
+## Let's start with some numbers
+
+It's been a nice and steady growth, given that our only marketing is Twitter, and of course all the love that we are receiving over there. We are reaching around **60K visits** per month and the grand total of the visits in our first year is well over **500K**!
+
+### Visits
+
+![Weekly visists](ga-users.png)
+
+### Commits
+
+We have well over [100 contributors](https://github.com/excalidraw/excalidraw/graphs/contributors) to the project and we have almost daily incremental improvments and deployments to production. Everything is updated frequntly and automatically with tools like [Vercel](https://vercel.com/), [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/), [Crowdin](https://crowdin.com/project/excalidraw), and others.
```

Comment:
```suggestion
We have over [100 contributors](https://github.com/excalidraw/excalidraw/graphs/contributors) that help us ship almost daily fixes, improvements, and new features right to production. Using tools and services like [Vercel](https://vercel.com/), [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/), [Crowdin](https://crowdin.com/project/excalidraw), and others, we can to smooth this process out, or completely automate it.
```

Fixed the typos, removed repetition ("well over"), tweaked some sentences. Subjective of course, feel free to pick & choose.